tests/stats/mda8/test_mda8.py

```diff
@@ -26,13 +27,13 @@ def test_data(time, values) -> xr.DataArray:
         pytest.param(
             xr.date_range(start="2024-01-01 01:00", periods=49, freq="1h"),
             [0.0] * 49,
-            [0, 0, np.nan],
+            [np.nan, 0, np.nan],
```

Comment:
why is the MDA8 for the 1st day nan?
The 8-hour mean needs 6 or more hours to compute, so the 1st day has 19 8-hour means: 01-06, 01-07, 01-08, ..., 17-24

Reply:
1-9, 2-10, 3-11, 4-12, 5-13, (<- These are NaN because of 6/8 required for 8h average), 6-14, 7-15, 8-16, 9-17, 10-18, 11-19, 12-20, 13-21, 14-22, 15-23, 16-24, (Next day ->), 17-1...

I count 11 observations. Is this wrong?